### ISSUE TITLE
tagging-label does not seem to work.

### ISSUE THREAD

**Issue by RobiFerentz** [angular-ui/ui-select#458]
According to the wiki, tagging-label should add a string next to a potentially new item in the drop down but I've tried not setting it, which is supposed to default to '(new)' I've tried setting it with a literal and with expressions with and without {{}} and it still doesn't seem to work. Am I missing something?

**Comment by idanb11** [comment]
:+1:  me too...

**Comment by elutz** [comment]
The tagging label works only if you use an array of strings for the options. If you use an array of objects it fails. I looked at the code but to be honest I don't understand what is really going on there: `ui-select/dist/select.js:863` resp. `ui-select/src/uiSelectMultipleDirective.js:273`

**Comment by jashmenn** [comment]
This is true - it's actually broken even in the examples (e.g. [this one](https://github.com/angular-ui/ui-select/blob/master/examples/demo-tagging.html#L119)).

The reason is because the code that uses the `taggingLabel` in [uiSelectMultipleDirective.js](https://github.com/angular-ui/ui-select/blob/master/src/uiSelectMultipleDirective.js#L259) isn't implemented in [uiSelectSingleDirective.js](https://github.com/angular-ui/ui-select/blob/master/src/uiSelectSingleDirective.js#L109).

So even though it is possible to have a new item created when you hit Return, the code for showing a new item in the list doesn't seem to be implemented.

**Comment by KylePDavis** [comment]
:+1: Same issue for me in newer releases.

In the meantime though I have been able to work around this by using the v0.9.9 release with `$select.taggingLabel` in my markup.

e.g.,
```html
<i ng-if="item.isTag">{{$select.taggingLabel}}</i>
```

**Comment by YardGnomeNinja** [comment]
Thanks KylePDavis! Your work around works wonderfully!

**Comment by wesleycho** [comment]
Can some reproduce this in a Plunker based on the latest release so that this is clearer to understand?
